apps/protailwind/.env.production

```diff
@@ -10,9 +10,12 @@ NEXT_PUBLIC_PRODUCT_DESCRIPTION="Increase development velocity and craft sustain
 NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM=3179174
 NEXT_PUBLIC_CONVERTKIT_TOKEN=Icq6W1wbib8ey1TBlwlo9A
 NEXT_PUBLIC_CONVERTKIT_SUBSCRIBER_KEY=ck_subscriber_id
+NEXT_PUBLIC_CONVERTKIT_SUBSCRIBE_URL=/api/skill/subscribe/convertkit
+NEXT_PUBLIC_CONVERTKIT_GET_SUBSCRIBER_URL=/api/skill/subscriber/convertkit
 CONVERTKIT_BASE_URL=https://api.convertkit.com/v3/
 NEXT_PUBLIC_GOOGLE_ANALYTICS=CHANGE_ME
 NEXT_PUBLIC_OG_IMAGE_URI=https://protailwind-images.vercel.app/api/card
 NEXT_PUBLIC_SANITY_PROJECT_ID=knx644y7
 NEXT_PUBLIC_SANITY_DATASET_ID=production
-NEXT_PUBLIC_SANITY_API_VERSION="2022-07-29"
\ No newline at end of file
+NEXT_PUBLIC_SANITY_API_VERSION="2022-07-29"
+CASTINGWORDS_TRANSCRIPT_ORDERING_ENABLED=false
```

Comment:
the reason I strongly dislike this and the "lets generate AI transcripts on our own machines" default is that it effectively means we can't have an integrated process so the creator can upload a tip etc and do the work for us.

I'd strongly prefer that we let castingwords automate this work until we have a proper automation solution